src/Main.java

```diff
@@ -5,6 +5,11 @@ public static void main(String[] args) {
         for (int i = 0; i <= 100 ; i++) {
             System.out.println("TIC");
         }
+
+        for (int i = 0; i < 100; i++) {
+            System.out.println("TAC");
+        }
+
```

Comment:
This behaviour is wrong: The application should print one time TIC, one time TAC, and so on, times 100